jdi-light/src/main/java/com/epam/jdi/light/settings/DriverSettings.java

```diff
@@ -18,6 +18,7 @@
 import static org.openqa.selenium.PageLoadStrategy.NORMAL;
 
 public class DriverSettings {
+    public String type = "default"; //local/remote/default
```

Comment:
change to
public boolean remoteRun;

Reply:
Hi Roman!

Although I agree that using boolean property seems more logical here, there is a concern that we should take into account:

Existing projects don't have remoteRun property in their .properties files. Once someone upgrades to latest version of JDI, he'll have to specify this property or otherwise it'll take 'false' and change the existing behavior.

Benefit of proposed solution is that when property is not specified, it takes 'default' value and does not affect the original logic. So engineer can use both approaches: either switching between remote and local driver by enabling and disabling driver.remoteUrl or by explicitly specifying driver.type

Reply:
At the moment solution is following:
DRIVER.remoteRun is now boolean,
however, if driver.remote.run property is not explicitly set - original logic will be used.